docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -22,7 +22,7 @@ Sycamore is deployed using Docker, and you can launch it locally or on a virtual
 
 `Docker compose up --pull-always`
 
-Note: You can alternately remove the `--pull=always` and instead run docker compose pull to control when new images are downloaded. `--pull=always` guarantees you have the most recent images for the specified version.
+Note: You can alternately remove the `--pull=always` and instead run docker compose pull to control when new images are downloaded. `--pull=always` guarantees you have the most recent images for the specified version. 
```

Comment:
Remove spurious space added to end of line. It will make our linting unhappy.

Reply:
Done